packages/allocator/src/lablink_allocator_service/terraform/user_data.sh

```diff
@@ -131,15 +131,27 @@ if [ "$HAS_GPU" = true ]; then
     DOCKER_GPU_ARGS="--runtime=nvidia --gpus all"
 fi
 
+echo "> Creating config directory…"
+sudo mkdir -p /etc/config
+
+cat <<'EOF' > /etc/config/startup.sh
+${startup_content}
+EOF
+
+chmod +x /etc/config/startup.sh
```

Comment:
I am a bit confused with the similar names for the files, double check that these file names are correct, let's try to have consistent file naming schemes (start.sh for Docker entrypoint, user_data.sh for terraform starter scripts for VMs, and custom_start.sh for the custom script)